awx/ui_next/src/screens/Setting/MiscSystem/MiscSystemDetail/MiscSystemDetail.jsx

```diff
@@ -20,6 +20,25 @@ function MiscSystemDetail({ i18n }) {
   const { me } = useConfig();
   const { GET: allOptions } = useSettings();
 
+  const [executionEnvironmentId, setExecutionEnvironmentId] = useState('');
+
+  const {
+    isLoading: isLoadingExecutionEnvironment,
+    error: errorExecutionEnvironment,
+    request: fetchExecutionEnvironment,
+    result: executionEnvironment,
+  } = useRequest(
+    useCallback(async () => {
+      if (!executionEnvironmentId) {
+        return '';
+      }
+      const { data } = await ExecutionEnvironmentsAPI.readDetail(
+        executionEnvironmentId
+      );
+      return data;
+    }, [executionEnvironmentId])
+  );
+
```

Comment:
Since the settings API does not have `summary_fields`. It was necessary an extra request for Details/Edit screens in order to fetch the EE and display the proper name.

Reply:
Also, I did not add a `Revert` for the EE lookup.